libc/src/__support/fixedvector.h

```diff
@@ -63,6 +63,10 @@ template <typename T, size_t CAPACITY> class FixedVector {
     return reverse_iterator{&store[item_count]};
   }
   LIBC_INLINE constexpr reverse_iterator rend() { return store.rend(); }
+
+  using iterator = typename cpp::array<T, CAPACITY>::iterator;
+  LIBC_INLINE constexpr iterator begin() { return store.begin(); }
+  LIBC_INLINE constexpr iterator end() { return iterator{&store[item_count]}; }
```

Comment:
Does returning `store.end()` not work?
```suggestion
  LIBC_INLINE constexpr iterator end() { return store.end(); }
```

Reply:
Ah, that makes sense. I'm wondering if something like
`return store.begin() + item_count` would work better.

Reply:
Would that account for the size of the objects themselves when doing the pointer arithmetic? i.e. if an int is 4 bytes and you have 4 elements would you want `end()` to be at `store.begin() + 4*4`?

Reply:
Gotcha, I was waiting on rebuilding LLVM this morning but I can try it when I get home later today

Reply:
I actually prefer `&store[item_count]` and let the compiler do the pointer arithmetic for you.

Reply:
Just using the iterator directly makes sure it's the correct iterator type though.